src/components/MovieList/MovieList.jsx

```diff
@@ -8,7 +9,9 @@ const MovieList = ({ movies }) => {
       <h2 className="movie-list__heading">Animations Movies</h2>
       <div className="movie-list__items">
         {movies.map((movie) => (
-          <MovieCard key={movie.id} movie={movie} />
+          <Link key={movie.id} to={`/movie/${movie.id}`}>
```

Comment:
Nice job linking the path to details page!!

Reply:
Thank you!!